stories/calendar/calendar.stories.js

```diff
@@ -288,7 +309,8 @@ export const days = () => `
 
 days.parameters = {
     docs: {
-        iframeHeight: 400
+        iframeHeight: 400,
+        storyDescription: 'The calendar component can display days of the month by adding the <code>–days</code> modifier class to the element.'
```

Comment:
I think here you mean "--dates" on the content element, "fd-calendar__content--dates" correct?

Reply:
@mikerodonnell89  That's correct, if I added **"to the <code>fd calendar__content</code> element"** is that more understandable? Or can I simply add **"content element"** at the end of the sentence?

Reply:
I just updated the sentences, let me know if they make sense now :)